src/erlcloud_mon.erl

```diff
@@ -216,10 +217,51 @@ put_metric_data(Namespace, MetricName, Value, Unit, Timestamp) ->
          ),
     mon_simple_query(Config, "PutMetricData", Params).
 
+
+%%------------------------------------------------------------------------------
+%% @doc CloudWatch API - GetMetricStatistics - Easy average version
+%% Gets average stats at 60 second intervals for the given metric on the given instance for the given interval
+%% @end
+%%------------------------------------------------------------------------------
+-spec get_metric_statistics(
+        MetricName  ::string(),
+        StartTime   ::string(),
```

Comment:
The example doesn't show this as being a string? One or the other should be fixed.